cert/aziot-certd-config/src/lib.rs

```diff
@@ -358,6 +365,7 @@ mod tests {
     fn parse_config() {
         let actual = r#"
 homedir_path = "/var/lib/aziot/certd"
+max_requests = 50
```

Comment:
Oh.. so now we need to make the limit 50 so that EdgeAgent's requests succeed?

Reply:
This is test code. The default remains 10.

Reply:
ah, yes, missed that.